File: sources/icloud-workspace/seis-digital-experience-foundation/runtime/server-upload-bundle/source/polyglot/go/readiness_signal.go
package main

type ReadinessSignal struct {
	Lane   string `json:"lane"`
	Status string `json:"status"`
	Score  int    `json:"score"`
}

func BuildReadinessSignal() ReadinessSignal {
	return ReadinessSignal{Lane: "go", Status: "reference", Score: 72}
}

File: sources/icloud-workspace/seis-digital-experience-foundation/runtime/server-upload-bundle/source/polyglot/rust/readiness_signal.rs
pub struct ReadinessSignal {
    pub lane: &'static str,
    pub status: &'static str,
    pub score: u8,
}

pub fn build_readiness_signal() -> ReadinessSignal {
    ReadinessSignal {
        lane: "rust",
        status: "reference",
        score: 72,
    }
}
